confident_metrics/__init__.py

```diff
@@ -1,3 +1,2 @@
 """Python abstraction for runtime metrics collection built on top of Prometheus."""
-
-__version__ = "0.0.1"
+from confident_metrics.metrics import record_event  # noqa: F401
```

Comment:
Let's leave the version here, following the PEP <number is missing>

Reply:
If I add it here I get an error while loading the file in setup.py.

Does it really make sense not making the only function in the package available from the the package namespace itself?

shouldn't `from confident_metrics import record_event` be the canonical way to import it? It looks to me way more pythonic than `from confixdent_metrics.metrics import record_event`.

Is there any other way to specify the version requirements without screwing up the namespace alias? Should I surround the import with a try-except?

Reply:
nvm, I put a try-except like you do in ml-core

Reply:
I am not saying we should change your `record_event` import, I am saying that we need to have `__version__` :)